crates/holochain_zome_types/src/record/facts.rs

```diff
@@ -65,7 +65,7 @@ mod tests {
         let hn = not_(action_facts::is_new_entry_action()).build(u);
         let mut he = action_facts::is_new_entry_action().build(u);
         *he.entry_data_mut().unwrap().0 = EntryHash::with_data_sync(&e);
-        let he = Action::from(he);
+        let he = he;
```

Comment:
This probably does nothing =D

Reply:
okay, I mean before it looked like it was constructing an Action, now it just makes it immutable. In any case, `he` isn't used afterwards.